docs/DeveloperGuide.adoc

```diff
@@ -640,8 +733,7 @@ public static TaskList searchTaskList(String listTitle) throws CommandException
 
 ===== Aspect: Format of add default task list command
 
-* **Alternative 1 (current choice):** only one command word, without any other argument
-** Pros: Less parser files to manage altogether. Very simple and easy to type.
+* There is only one command word, without any other argument. Therefore, less parser files are needed to manage altogether. This is very simple and easy to type.
```

Comment:
this is not put in a table as the rest?

Reply:
This one only has one choice, so I am considering to rephrase it into one bullet point